RELEASE.md

```diff
@@ -32,14 +32,25 @@ automated and require no human action.
   completed
 6. (optional) If a fix is required to the release-candidate:
   1. Merge the fix with `master` first
-  2. Checkout the release-candidate branch and merge `master`
-  3. Revert all changes since the creation of the release-candidate that are
-  **not** required for the fix.
-  4. Push the release-candidate branch to Github - this is now the new release-
+  2. Cherry-pick the commit from `master` to `release-v0.8.26`, fixing any
```

Comment:
```suggestion
  2. Cherry-pick the commit from `master` to `release-vVERSION`, fixing any
```

Reply:
I prefer to use an example version string in docs over some placeholder value (like `VERSION`) because I feel it makes the examples clearer.